src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,52 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1
+    def test_0019_bz2070189(self, multihost, adjoin):
+        """
+        :title: sssd error triggers backtrace :
+        [write_krb5info_file_from_fo_server] (0x0020): [RID#73501]
+        There is no server that can be written into kdc info file. is set
+        and files provider is also enabled
```

Comment:
Are the above 3 lines part of the title? The can be added to `:description:` 
Indent the lines to start from under `d` of description or `t` of title or it will not be picked up by the importer

Reply:
Maybe use that in the description and something like Sumit's comment in the PR for the title?

"Global Catalog lookups should not update the kdcinfo files"